peg/Parser.hx

```diff
@@ -337,6 +359,45 @@ class Parser {
 		return fn;
 	}
 
+	function parseDocTagReturnType(fn:PFunction):Null<PType> {
+		var documentedType:Null<String> = null;
+
+		if (docTagReturnTypeRE.match(fn.doc)) {
+			var type = docTagReturnTypeRE.matched(1);
+			if (combinationTypeRE.match(type)) {
+				type = 'mixed';
```

Comment:
Please, add a `//TODO` to not forget to implement correct parsing of `type1|type2`

Reply:
done in 09f1f4c